slsa-provenance/main.go

```diff
@@ -116,12 +120,14 @@ func main() {
 			panic(err)
 		}
 
-		fmt.Printf(string(attBytes))
-		fmt.Printf(`::set-output name=provenance::%s`, string(attBytes))
+		filename := fmt.Sprintf("%s.intoto", binary)
```

Comment:
is there any sort of verification in the filename we need to perform? The builder's dry run should be validating it already, ie that it only contains alphanumeric characters. Add a comment. How about re-implementing the verification for defense in depth?

Reply:
done